scripts/openapi_generator/app.py

```diff
@@ -75,15 +77,27 @@ def create_llama_stack_app() -> FastAPI:
         ],
     )
 
-    # Get all API routes
+    # Include routers for APIs that have them
+    protocols = api_protocol_map()
+    for api in protocols.keys():
+        # For OpenAPI generation, we don't need a real implementation
+        router = build_fastapi_router(api, None)
+        if router:
+            app.include_router(router)
+
+    # Get all API routes (for legacy webmethod-based routes)
     from llama_stack.core.server.routes import get_all_api_routes
 
     api_routes = get_all_api_routes()
 
-    # Create FastAPI routes from the discovered routes
+    # Create FastAPI routes from the discovered routes (skip APIs that have routers)
     from . import endpoints
 
     for api, routes in api_routes.items():
+        # Skip APIs that have routers - they're already included above
+        if build_fastapi_router(api, None) is not None:
```

Comment:
another nitpick, but having a dict or fastapi routers above where you "build" them and then just checking against that would be better. even though this code is harmless right now, I am just suspicious of things like `build_` becoming heavier over time.